packages/client/components/SnackbarMessage.tsx

```diff
@@ -88,15 +88,24 @@ const SnackbarMessage = (props: Props) => {
     onMouseLeave
   } = props
   useTransitionEnd(300, status, onTransitionEnd)
+  //Currently do not need dismiss button
+  const displayDismissButton = false
```

Comment:
It is now hardcoded for all snackbars. What was meant is that we want to be able to enable this close button individually per snackbar.

So can we make this option to be possible to specify like this: 

```
atmosphere.eventEmitter.emit('addSnackbar', {
        autoDismiss: 5,
        message: 'Hello world,
        showDismissButton: true,
        
``` 

?

Reply:
Ok

Reply:
I hope I covered all snackbars.